sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -657,6 +659,20 @@ class Analyzer(
     }
   }
 
+  /**
+   * Resolve table relations with concrete relations from v2 catalog.
+   *
+   * [[ResolveRelations]] still resolves v1 tables.
+   */
+  object ResolveTables extends Rule[LogicalPlan] {
```

Comment:
Can we use `ResolveV2Relations` instead in order to avoid those confusion?
```scala
-   * Resolve table relations with concrete relations from v2 catalog.
-   *
-   * [[ResolveRelations]] still resolves v1 tables.
+   * Replaces [[UnresolvedRelation]]s with concrete relations from the v2 catalog.
    */
-  object ResolveTables extends Rule[LogicalPlan] {
+  object ResolveV2Relations extends Rule[LogicalPlan] {
```

Reply:
Then, the batch looks like
```scala
ResolveV2Relations ::
ResolveRelations ::
```
instead
```scala
ResolveTables ::
ResolveRelations ::
```

Reply:
Please ignore the above comment.

Reply:
Name it `ResolveTables` because there may be a new rule `ResolveViews` down the road, which will be part of ViewCatalog effort. More details to come.